src/shell/components/FieldTypeTinyMCE/index.tsx

```diff
@@ -0,0 +1,319 @@
+import React, { useEffect, useState } from "react";
+import { Editor } from "@tinymce/tinymce-react";
+import { Box, alpha } from "@mui/material";
+import { theme } from "@zesty-io/material";
+
+// TinyMCE so the global var exists
+import tinymce from "tinymce/tinymce";
+// DOM model
+import "tinymce/models/dom/model";
+// Theme
+import "tinymce/themes/silver";
+// Toolbar icons
+import "tinymce/icons/default";
+// Editor styles
+// import "tinymce/skins/ui/oxide/skin.min.css";
+import "tinymce/skins/ui/tinymce-5/skin.min.css";
+
+// importing the plugin js.
+// if you use a plugin that is not listed here the editor will fail to load
+import "tinymce/plugins/advlist";
+import "tinymce/plugins/anchor";
+import "tinymce/plugins/autolink";
+import "tinymce/plugins/autoresize";
+import "tinymce/plugins/autosave";
+import "tinymce/plugins/charmap";
+import "tinymce/plugins/code";
+import "tinymce/plugins/codesample";
+import "tinymce/plugins/directionality";
+import "tinymce/plugins/emoticons";
+import "tinymce/plugins/fullscreen";
+import "tinymce/plugins/help";
+import "tinymce/plugins/image";
+import "tinymce/plugins/importcss";
+import "tinymce/plugins/insertdatetime";
+import "tinymce/plugins/link";
+import "tinymce/plugins/lists";
+import "tinymce/plugins/media";
+import "tinymce/plugins/nonbreaking";
+import "tinymce/plugins/pagebreak";
+import "tinymce/plugins/preview";
+import "tinymce/plugins/quickbars";
+import "tinymce/plugins/save";
+import "tinymce/plugins/searchreplace";
+import "tinymce/plugins/table";
+import "tinymce/plugins/template";
+import "tinymce/plugins/visualblocks";
+import "tinymce/plugins/visualchars";
+import "tinymce/plugins/wordcount";
+import "./plugins/slashcommands";
+import "./plugins/socialmediaembed";
```

Comment:
Are we actually using all of these plugins or just including them because TinyMCE includes them? e.g. `autosave` do we have this tied into our system? If not how is this being used? If not being used are we shipping extra code that's adding to bloat?